wicket-core/src/test/java/org/apache/wicket/ajax/AjaxRequestHandlerTest.java

```diff
@@ -284,14 +283,14 @@ void ajaxRedirectSetsNoCachingHeaders()
 		tester.startPage(new Wicket3921());
 
 		tester.clickLink("updatePage");
-		assertEquals(Time.START_OF_UNIX_TIME.toRfc1123TimestampString(),
+		assertEquals(Instants.toRFC7231Format(Instant.EPOCH),
```

Comment:
Why change the RFC ?

Reply:
It seems the right RFC for Date formats is RFC7231:
https://tools.ietf.org/html/rfc7231#section-7.1.1.1

But I admit to be a little confused, especially in respect of what RFC1123 actually says about Date formats.